packages/admin-ui/src/app/participants/pending-approvals.component.html

```diff
@@ -0,0 +1,132 @@
+<div class="row mb-2">
+  <div class="col">
+    <h3>Pending Approval Requests</h3>
+  </div>
+</div>
+
+<ul ngbNav #nav="ngbNav" class="mt-4 nav-tabs" (navChange)="tabChange($event)">
+  <li ngbNavItem="fundAdjustment">
+    <!-- fundAdjustment -->
+    <a ngbNavLink
+      >Fund Adjustment
+      <span class="ml-2 badge bg-danger text-white">{{
+        fundAdjustmentCount
+      }}</span></a
+    >
+    <ng-template ngbNavContent>
+      <h4 class="mt-3">Pending Fund Adjustment for all Participants</h4>
+      <div>
+        <button class="btn btn-success" (click)="approvePendingApprovals()">
```

Comment:
These buttons should only be enabled if items are selected

Reply:
These buttons will approve/reject items in all tabs, not just the ones we're seeing in the current tab - this is misleading to the user.

Either the per tab buttons only affect the items in the current tab, or the approve/reject buttons should be moved to outside the tabs.